### PR TITLE
Tom Hanson - LeedsJS event added

### PR DESCRIPTION
**Pull request by tmhn**
*Description of changes:*

- Added Bio markdown file and avatar image
- Added LeedsJS event - I'll update closer to the time with a title and short bio of the talk.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by harrysolovay** [comment]
@tmhn you're missing a `'` to delineate the end of your bio. Can you please fix this. Otherwise, looks good 👍

File: content/contributors/tom-hanson/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Tom Hanson
+avatar: './avatar.jpg'
+bio: 'Software Developer at AND Digital
```
Comment (ahmadawais) [inline content/contributors/tom-hanson/index.md]:
```suggestion
bio: 'Software Developer at AND Digital'
```

**Comment by tmhn** [comment]
> @tmhn you're missing a `'` to delineate the end of your bio. Can you please fix this. Otherwise, looks good 👍

Good spot @harrysolovay thanks!

**Comment by swaminator** [comment]
@tmhn I responded to your email about the event. Will keep this open to track progress. Also, would be great to get on a call this week if you're available.

**Comment by swaminator** [comment]
@tmhn we've added an event description page (looks like https://amplify.aws/community/events/2019/08/01/jamstack). It would be great if you can add a description of what you're going to be presenting, and at what date and time. Thanks!

**Comment by swaminator** [comment]
@tmhn just following up here.

**Comment by harrysolovay** [comment]
@tmhn can you please sync your fork with the upstream so that the TravisCI config is present (which will result in the checks passing).

**Comment by swaminator** [comment]
Your event has been approved. You will be receiving a package in the mail soon.

**Pull request closed without merging**